code/modules/admin/verbs/randomverbs.dm

```diff
@@ -1079,234 +1079,23 @@ Traitors and the like can also be revived with the previous role mostly intact.
 	if(!check_rights(R_ADMIN) || !check_rights(R_FUN))
 		return
 
-	var/list/punishment_list = list(
-		ADMIN_PUNISHMENT_AHEAL,
-		ADMIN_PUNISHMENT_ANIMALIZE,
-		ADMIN_PUNISHMENT_BRAINDAMAGE,
-		ADMIN_PUNISHMENT_BSA,
-		ADMIN_PUNISHMENT_CLUWNE,
-		ADMIN_PUNISHMENT_CRYO,
-		ADMIN_PUNISHMENT_DAMAGE,
-		ADMIN_PUNISHMENT_DCHAT_ANARCHY,
-		ADMIN_PUNISHMENT_DCHAT_DEMOCRACY,
-		ADMIN_PUNISHMENT_FIREBALL,
-		ADMIN_PUNISHMENT_FORCESAY,
-		ADMIN_PUNISHMENT_GHOST,
-		ADMIN_PUNISHMENT_GIB,
-		ADMIN_PUNISHMENT_IMMERSE,
-		ADMIN_PUNISHMENT_LIGHTNING,
-		ADMIN_PUNISHMENT_PACIFISM,
-		ADMIN_PUNISHMENT_PUZZLE,
-		ADMIN_PUNISHMENT_ROD,
-		ADMIN_PUNISHMENT_SLEEP,
-		ADMIN_PUNISHMENT_SUPPLYPOD,
-		ADMIN_PUNISHMENT_SUPPLYPOD_QUICK,
-		ADMIN_PUNISHMENT_VOICE_OF_GOD
-	)
-	if(iscarbon(target))
-		punishment_list += ADMIN_PUNISHMENT_COOKIE
-		punishment_list += ADMIN_PUNISHMENT_NUGGET
-	if(ishuman(target))
-		punishment_list += ADMIN_PUNISHMENT_FLOORCLUWNE
-		punishment_list += ADMIN_PUNISHMENT_FLOORCLUWNE_STALKER
-		punishment_list += ADMIN_PUNISHMENT_STALKER
-		punishment_list += ADMIN_PUNISHMENT_TOE
-		punishment_list += ADMIN_PUNISHMENT_TOEPLUS
-
-	var/punishment = input("Choose a punishment", "DIVINE SMITING") as null|anything in sort_list(punishment_list)
+	var/punishment = input("Choose a punishment", "DIVINE SMITING") as null|anything in GLOB.smites
```

Comment:
since we have tgui list, would you change it into tgui list? That'll make admin easy to find a specific smite.

Reply:
done 0ba8b99